### ISSUE TITLE
Forced Dark navigation item focus ring not visible

### ISSUE THREAD

**Issue by shainanigans** [StackExchange/Stacks#995]
**Describe the bug**
In the forced Dark theme (like in Teams), the focus ring on `s-navigation--item` is not visible.

**To Reproduce**
1. Go to the reference documentation's Theming section and find the Dark theme.
2. Using your keyboard only, navigate to the "Products" link.
3. See that there is no visible focus ring on the link (it's there, just impossible to see).

**Expected behavior**
Focus ring is visible in the Dark theme. Ring should have 3:1 color contrast ratio with the dark background of the topbar in order to meet color contrast requirements for non-text elements.

**Screenshots**
<img width="163" alt="Screen Shot 2022-06-27 at 3 44 46 PM" src="https://user-images.githubusercontent.com/14816224/176023052-671464bc-dbcb-46ac-b0b7-26d1958d494a.png">
